infra/templates/az-isolated-service-single-region/ase.tf

```diff
@@ -55,8 +44,8 @@ module "app_service" {
   source                           = "../../modules/providers/azure/app-service"
   app_service_name                 = var.app_service_name
```

Comment:
Should this be app_service_name(s) instead? At first glance, it looks dedicated to a single application.

Reply:
I agree. This came up in a previous PR as well but I want to wait to make any changes to this until we pick up https://github.com/microsoft/cobalt/issues/196. This variable will need to be renamed and refactored to have a richer schema that allows us to define things like (1) git repo, (2) location of docker file, (3) image name and (4) image tag (along with other metadata most likely)

Based on this I think it makes sense to defer this work until we pick up https://github.com/microsoft/cobalt/issues/196 (most likely next week)